docs/reference/migration/migrate_5_0/search.asciidoc

```diff
@@ -128,6 +128,7 @@ in favour of `query` and `no_match_query`.
 
 * Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoPolygonQuery. Use parameter validation_method instead.
 
+* Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoDistanceRangeQuery. Use parameter validation_method instead.
```

Comment:
There only is a link to https://www.elastic.co/guide/en/elasticsearch/reference/master/query-dsl-geo-distance-range-query.html which has ignore_malformed explained, but not coerce. 

@clintongormley  need your feedback here: When deprecating parameters in the query dsl in favour of a new parameter - what's the best way to add deprecation warnings to the docs? I see it being done e.g. here https://www.elastic.co/guide/en/elasticsearch/reference/2.3/mapping-transform.html with some warning box, should we show something similar for query parameters, or is it enough to list those in the migration doc? Related to that, how do we deal with formerly valid but undocumented query parameters?